api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2277,6 +2344,19 @@ service AuthService {
   // DeleteAllSnowflakeSessions removes all Snowflake web sessions.
   rpc DeleteAllSnowflakeSessions(google.protobuf.Empty) returns (google.protobuf.Empty);
 
+  // CreateSAMLIdPSession creates web session with sub kind saml_idp used by the SAML IdP.
+  rpc CreateSAMLIdPSession(CreateSAMLIdPSessionRequest) returns (CreateSAMLIdPSessionResponse);
```

Comment:
Can we move this out from AuthService like we already do for LoginRules and DeviceTrust?